plugins/become/sudosu.py

```diff
@@ -55,6 +55,18 @@
             ini:
               - section: sudo_become_plugin
                 key: password
+        alt_method:
+            description: 
+              - Whether to use an alternative method to call sudosu. 
```

Comment:
This is about calling `su`, not `sudosu`, right? I would also add some explanation what it does.
```suggestion
              - Whether to use an alternative method to call C(su). Instead of running C(su -l user /path/to/shell -c command),
                it runs C(su -l user -c command).
```